src/pebble.py

```diff
@@ -75,7 +75,8 @@ def _get_pebble_layer(jenkins_instance: jenkins.Jenkins) -> ops.pebble.Layer:
                 "command": f"java -D{jenkins.SYSTEM_PROPERTY_HEADLESS} "
                 f"-D{jenkins.SYSTEM_PROPERTY_LOGGING} "
                 f"-jar {jenkins.EXECUTABLES_PATH}/jenkins.war "
-                f"--prefix={env_dict['JENKINS_PREFIX']}",
+                f"--prefix={env_dict['JENKINS_PREFIX']}"
+                "-XX:MaxRAMPercentage=50.0 -XX:InitialRAMPercentage=50.0",
```

Comment:
Why not a bigger value? 75?

Reply:
I discussed it with our team at it seems like any value above 50 can cause problems (ref: https://docs.cloudbees.com/docs/cloudbees-ci-kb/latest/best-practices/jvm-memory-settings-best-practice). So we'll go with 50 for now and I'll do some experimentation to see if we can make it higher.